src/gc/gc_alloc.h

```diff
@@ -77,7 +77,7 @@ extern "C" inline void* gc_alloc(size_t bytes, GCKind kind_id) {
 
     // TODO This doesn't belong here (probably in PythonGCObject?)...
     if (kind_id == GCKind::PYTHON) {
-        ((Box*)r)->cls = NULL;
+        memset(r, 0, bytes);
```

Comment:
Hmm I was hoping that we could avoid doing this.  I think in some places we explicitly zero out any extension-defined memory; is that something we could do here?